### DISCUSSION TITLE
AV1 codec support?

### DISCUSSION THREAD

**Discussion by unicornsoftwareinc** [ant-media/Ant-Media-Server#5616] [General]
With Apple finally starting to support hardware decode for the AV1 codec on new iPhones and hardware decode likely coming for macOS devices with M3 chips, it seems like AV1 is getting much closer to universal support. See AV1 support on caniuse: https://caniuse.com/?search=AV1

AV1 codec support means potentially big quality improvements for our users in lower bandwidth/poor network environments. It would really be great to see Ant Media support this feature in the not too distant future.

Is there any update from the Ant Media team on if/when this might be coming?

Thanks!

**Comment by Mohit-3196** [comment]
Hi @unicornsoftwareinc,
How are you?

Thank you for the updates about the latest advancements and the prospects.
AV1 codec support is in our roadmap for sure with the WebRTC stack update and I think it should be available soon.

Let me invite our lead developer @mekya to comment on this and the road ahead.

Thank you

**Comment by unicornsoftwareinc** [comment]
Glad to hear it is on the roadmap. Any estimate in terms of time would be appreciated!

**Reply by yashtandon113** [reply]
Hi @unicornsoftwareinc 

The ETA for the implementation of AV1 video codec is Q1 of 2024.

Currently, we haven't received a significant number of requests from our users and customers for this feature, so we've made the decision to delay its implementation until next year. If we notice an increase in requests in the coming months, we're open to revisiting the timeline and potentially giving it higher priority. 

Right now, our primary focus is on improving the reliability and performance of our existing features, as well as making them more robust.

Please let us know if you have any questions. We are open for the discussion.

**Comment by unicornsoftwareinc** [comment]
Hi @yashtandon113 glad to hear. Just wanted to mention that the latest Macs with M3 chips do indeed have hardware decode support built-in now. So really would be awesome if you guys manage to get this feature out in Q1. Thanks!
